components/bookmark/index.test.jsx

```diff
@@ -41,7 +42,65 @@ describe("Bookmark", () => {
     );
     expect(asFragment()).toMatchSnapshot();
   });
+
+  test("it renders a bookmark icon and text if withText prop is true", async () => {
+    const { asFragment } = renderWithTheme(
+      <BookmarkContext.Provider value={(bookmarks, setBookmarks)}>
+        <Bookmark iri={iri} withText />
+      </BookmarkContext.Provider>
+    );
+    const text = screen.getByTestId("bookmark-text");
```

Comment:
I think you can might as well get `getByTestId` from the resulting object from `renderWithTheme` (e.g. `const { asFragment, getByTestId }`. Don't think there's much difference, but then you have to import one less dependency.